Materials2050_oM/Fragments/ResultsNavigation.cs

```diff
@@ -20,30 +20,32 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
+using System.ComponentModel;
 using BH.oM.Base;
-using BH.oM.Base.Attributes;
-using System;
 using System.Collections.Generic;
-using System.ComponentModel;
-using System.Linq;
 
-namespace BH.oM.Adapters.Materials2050
+namespace BH.oM.Materials2050.Fragments
 {
-    [Description("Object description in here. Will appear in the UI tooltip.")]
-    public class ExampleObject : BHoMObject
+    [Description("A data fragment containing API-specific values to help navigate paginated results from the GET request.")]
+    public class ResultsNavigation : IFragment
```

Comment:
Could also call it `ResultsPagination` to make it explicit to how the navigation is done? Suggestion only

Reply:
Yes I bounced around between that and the existing name. I'll try this out and see :)